packages/node_modules/@ciscospark/plugin-avatar/test/unit/spec/avatar.js

```diff
@@ -446,6 +424,112 @@ describe(`plugin-avatar`, () => {
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa3`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa3`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa4`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa4`),
+
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa0`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa0`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa0`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa0`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa1`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa1`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa2`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa2`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa4`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa4`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa4`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa4`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa4`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa4`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa1`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa1`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa2`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa2`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa2`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa2`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa3`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa3`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa1`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa1`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa4`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa4`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa4`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa4`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa0`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa0`),
+          assert.becomes(avatar.retrieveAvatarUrl(`88888888-4444-4444-4444-aaaaaaaaaaa5`), `https://example.com/88888888-4444-4444-4444-aaaaaaaaaaa5`)
+        ])
+          .then(() => {
+            assert.callCount(spark.request, 1);
+          });
+      });
```

Comment:
This test is simply a workout of duplicate handling. See general comment on proof of fix wrt batcher returning incorrect values